tests/acceptance/features/coreApiVersions/fileVersionAuthor.feature

```diff
@@ -7,7 +7,6 @@ Feature: file versions remember the author of each version
     And user "Alice" has been created with default attributes and without skeleton files
     And user "Brian" has been created with default attributes and without skeleton files
     And user "Carol" has been created with default attributes and without skeleton files
-    And the administrator has enabled the file version storage feature
```

Comment:
Are any features from this file relevant for ocis? Will they be implemented in future?

Reply:
Just for future reference If we remove this then we might also need to change the title of scenarios as they have `enable file versioning and ......` but not in this PR

Reply:
The version author feature is not available in ocis, as per this comment https://github.com/owncloud/core/pull/40538#issuecomment-1384891184 we are keeping the tests in ocis. What's your thought @saw-jan ?

Reply:
IMO, we keep it.
If the version author is available by default in oCIS then I think the test will catch it